resources/tests.mjs

```diff
@@ -415,37 +394,38 @@ Suites.push({
 Suites.push({
     name: 'Inferno-TodoMVC',
     url: 'todomvc/architecture-examples/inferno/index.html',
-    prepare(runner, contentWindow, contentDocument) {
-        return runner.waitForElement('.new-todo').then(element => {
-            element.focus();
-            return element;
-        });
+    async prepare(page) {
+        const element = await page.waitForElement('.new-todo');
+        element.focus();
     },
     tests: [
-        new BenchmarkTestStep('Adding' + numberOfItemsToAdd + 'Items', (newTodo, contentWindow, contentDocument) => {
+        new BenchmarkTestStep('Adding' + numberOfItemsToAdd + 'Items', page => {
+            const newTodo = page.querySelector(".new-todo");
             for (let i = 0; i < numberOfItemsToAdd; i++) {
                 newTodo.value = 'Something to do ' + i;
                 newTodo.dispatchEvent(new Event('change', {
                   bubbles: true,
                   cancelable: true
                 }));
-                triggerEnter(newTodo, 'keydown');
+                page.triggerEnter(newTodo, 'keydown');
             }
         }),
-        new BenchmarkTestStep('CompletingAllItems', (params, contentWindow, contentDocument) => {
-            const checkboxes = contentDocument.querySelectorAll('.toggle');
+        new BenchmarkTestStep('CompletingAllItems', page => {
+            const checkboxes = page.querySelectorAll('.toggle');
             for (let i = 0; i < checkboxes.length; i++)
                 checkboxes[i].click();
         }),
-        new BenchmarkTestStep('DeletingItems', (params, contentWindow, contentDocument) => {
-            const deleteButtons = contentDocument.querySelectorAll('.destroy');
+        new BenchmarkTestStep('DeletingItems', page => {
+            const deleteButtons = page.querySelectorAll('.destroy');
             for (let i = 0; i < numberOfItemsToAdd; i++)
-                contentDocument.querySelector('.destroy').click();
+                page.querySelector('.destroy').click();
```

Comment:
Huh, it seems unnecessary to keep calling querySelector if we to all the buttons to delete in the first place.
This is unrelated to your change so we can fix it up in a separate PR.